static-site/app/not-found.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import Splash from "../components/splash";
+
+import styles from "./styles/not-found.module.css";
+
+export default function FourOhFour(): React.ReactElement {
+  return (
+    <>
+      <div className={styles.errorContainer}>
+        Oops - that page doesn’t exist! (404).
+        <br />
+        Here’s the splash page instead…
+      </div>
+
+      <Splash />
```

Comment:
Todo - drop this, as it's a placeholder (literally!) while the splash page is still part of the pages router.

Reply:
this is actually already fixed in the [prior PR](https://github.com/nextstrain/nextstrain.org/pull/1032); once that is merged, this PR will get rebased on top of it and will pick up [that change](https://github.com/nextstrain/nextstrain.org/pull/1032/commits/4f649994c05ace441e03e42f4fe88c5b0a367c8f).